adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryExecParamRepository.java

```diff
@@ -269,24 +275,10 @@ public Options getOptions() {
         // define options related to GTFS file `pathways.txt`
         options.addOption(String.valueOf(EXCLUSION_KEY.charAt(1)), EXCLUSION_KEY, true,
                 "Exclude files from semantic GTFS validation");
-        options.addOption(PATHWAY_MIN_LENGTH_KEY, PATHWAY_MIN_LENGTH_KEY, true,
```

Comment:
Why were all those removed?

Reply:
They were not used actually, I'll bring them back when implementing semantic validation of field pathways.length.

Reply:
Replying to https://github.com/MobilityData/gtfs-validator/pull/297#discussion_r455870497: will do so.